### ISSUE TITLE
if output dir contains dot(.), error comes

### ISSUE THREAD

**Issue by caihuach** [javascript-obfuscator/javascript-obfuscator#557]
for 0.24.6, if output dir has dot like a.b, it shows `Output path for directory obfuscation should be a directory path`
0.24.5 has no problem
thank you!

## Expected Behavior

normal output to any dirname

## Current Behavior

cannot output to dirname containing dot

## Steps to Reproduce

1. npx javascript-obfuscator@0.24.6 ./dist --output a.b
2. Output path for directory obfuscation should be a directory path
3. npx javascript-obfuscator@0.24.6 ./dist --output ab
4. Good

## Your Environment

* Obfuscator version used: 0.24.6
* Node version used: 10.18.1

# Stack trace

/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfuscator/dist/webpack:/JavaScriptObfuscator/src/cli/utils/ObfuscatedCodeWriter.ts:41                                                                     throw new Error('Output path for directory obfuscation should be a directory path');                          ^                                                                                 Error: Output path for directory obfuscation should be a directory path                                     at t.ObfuscatedCodeWriter.getOutputCodePath (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfuscator/dist/webpack:/JavaScriptObfuscator/src/cli/utils/ObfuscatedCodeWriter.ts:41:23)    at forEach (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfuscator/dist/webpack:/JavaScriptObfuscator/src/cli/JavaScriptObfuscatorCLI.ts:141:62)                                       at Array.forEach (<anonymous>)                                                                          at _.apply (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfuscator/dist/we$pack:/JavaScriptObfuscator/src/cli/JavaScriptObfuscatorCLI.ts:140:24)                                       at _.processSourceCodeData (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-ob$uscator/dist/webpack:/JavaScriptObfuscator/src/decorators/Initializable.ts:60:39)                           at _.apply (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfuscator/dist/we$pack:/JavaScriptObfuscator/src/cli/JavaScriptObfuscatorCLI.ts:78:14)                                        at _.run (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfuscator/dist/webp$ck:/JavaScriptObfuscator/src/decorators/Initializable.ts:60:39)                                             at Function.obfuscate (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfusca$or/dist/webpack:/JavaScriptObfuscator/src/JavaScriptObfuscatorCLIFacade.ts:9:33)                            at Object.<anonymous> (/home/caihua/.nvm/versions/node/v10.18.1/lib/node_modules/javascript-obfusca$or/bin/javascript-obfuscator:3:30)                                                                          at Module._compile (internal/modules/cjs/loader.js:778:30)                                           

## Minimal working example that will help to reproduce issue

```bash
npx javascript-obfuscator@0.24.6 ./dist --output a.b # bad
npx javascript-obfuscator@0.24.5 ./dist --output a.b # good
```

**Comment by sanex3339** [comment]
Will be fixed here:
https://github.com/javascript-obfuscator/javascript-obfuscator/pull/552

**Comment by sanex3339** [comment]
Released as `0.25.0`
